recipes/scons/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+import os
+import shutil
+import sys
+
+
+class SConsConan(ConanFile):
+    name = "scons"
+    description = "SCons is an Open Source software construction tool—that is, a next-generation build tool"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index/"
+    homepage = "https://scons.org"
+    topics = ("conan", "scons", "build", "configuration", "development")
+    settings = "os_build"  # Added to test on let CI test on all os'es
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("scons-{}".format(self.version), self._source_subfolder)
+
+    def build(self):
+        with tools.chdir(self._source_subfolder):
+            self.run("{} setup.py build -j{}".format(sys.executable, tools.cpu_count()))
+
+    def package(self):
+        self.copy("LICENSE*", src=self._source_subfolder, dst="licenses")
+
+        # Mislead CI and create an empty header in the includue directory
+        include_dir = os.path.join(self.package_folder, "include")
```

Comment:
maybe just put an exception to the hook's whitelist?

Reply:
@uilianries Is this workaround ok? Or does it need to be added to the hook whitelist?

Reply:
okay, let's leave it for now as is